### DISCUSSION TITLE
Can't connect to IBMi

### DISCUSSION THREAD

**Discussion by MichaelWerner** [codefori/vscode-ibmi#634] [IBM i] [answered]
Hello,

I downloaded Code for IBMi from the marketplace just now. When I try to connect to IBMi, I get this message 
`Max identification string size exceeded`
or this message
`ECONNREFUSED`

What can I do?

Thanks,

Michael

**Comment by worksofliam** [comment, marked as answer]
Hi @MichaelWerner 

* check SSHD is running
* check user `QSSHD` is enabled
* make sure you specified the correct SSH port. `22` is the default.
